### DISCUSSION TITLE
How do you get a list of all roles/permissions?

### DISCUSSION THREAD

**Discussion by christrain91** [clerk/javascript#2916] [Support request - DO NOT OPEN HERE]
Is there a way to get a list of roles/permissions from within the clerkClient?

**Comment by jeremy-cyberactive** [comment]
Looking at the pull request history, I can see these two PRs that were successfully merged back on November 22, 2023. They appear to add full CRUD support for Organization Roles and Permissions using the clerkClient, including listing roles and permissions.

https://github.com/clerk/javascript/pull/2177
https://github.com/clerk/javascript/pull/2178

However, these were both reverted on December 4, 2023:
https://github.com/clerk/javascript/pull/2252

The associated change notes do not give an explanation as to why that functionality was removed. It simply states 

> Drop the introduction of OrganizationRole and OrganizationPermission resources fro BAPI.

https://github.com/clerk/javascript/pull/2238

I'm curious as to why this functionality was removed. I'd like to build out a custom RBAC admin screen within my app, but it doesn't appear possible without those changes. Maybe @panteliselef can help shed some light?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
